scenarios/mono.benchmarks.yml

```diff
@@ -0,0 +1,299 @@
+      useMonoRuntime: jit
+      buildArguments: 
+        - "/p:IsDatabase=true"
+      environmentVariables:
+        database: PostgreSQL
+        connectionstring: Server={{databaseServer}};Database=hello_world;User Id=benchmarkdbuser;Password=benchmarkdbpass;Maximum Pool Size=256;NoResetOnClose=true;Enlist=false;Max Auto Prepare=4
+    load:
+      job: wrk
+      variables:
+        presetHeaders: json
+        path: /queries/queries=20
+  
+  updates-mono-jit:
+    db:
+      job: postgresql
+    application:
+      job: platformbenchmarks
+      useMonoRuntime: jit
+      buildArguments: 
+        - "/p:IsDatabase=true"
+      environmentVariables:
+        database: PostgreSQL
+        connectionString: Server={{databaseServer}};Database=hello_world;User Id=benchmarkdbuser;Password=benchmarkdbpass;Maximum Pool Size=256;NoResetOnClose=true;Enlist=false;Max Auto Prepare=4
+    load:
+      job: wrk
+      variables:
+        presetHeaders: json
+        path: /updates/queries=20
+
+profiles:
+  aspnet-citrine-lin:
+    variables:
+      serverPort: 5000
+      serverAddress: 10.0.0.102
+      cores: 28
+    jobs: 
+      db:
+        endpoints: 
+          - http://asp-citrine-db:5001
+      application:
+        endpoints: 
+          - http://asp-citrine-lin:5001
+        variables:
+          databaseServer: 10.0.0.103
+      load:
+        endpoints: 
+          - http://asp-citrine-load:5001
+  
+  aspnet-citrine-arm:
+    variables:
+      serverPort: 5000
+      serverAddress: 10.0.0.107
+      cores: 32
+    jobs: 
+      db:
+        endpoints: 
+          - http://asp-citrine-db:5001
+      application:
+        endpoints: 
+          - http://asp-citrine-arm:5001
+        variables:
+          databaseServer: 10.0.0.103
+      load:
+        endpoints: 
+          - http://asp-citrine-load:5001
+  
+  aspnet-citrine-amd:
+    variables:
+      serverPort: 5000
+      serverAddress: 10.0.0.103
+      cores: 48
+    jobs: 
+      db:
+        endpoints: 
+          - http://asp-citrine-db:5001
+      application:
+        endpoints: 
+          - http://asp-citrine-amd:5001
+        variables:
+          databaseServer: 10.0.0.103
+      load:
+        endpoints: 
+          - http://asp-citrine-load:5001
+  
+  aspnet-perf-lin:
+    variables:
+      serverPort: 5000
+      serverAddress: 10.0.0.102
+      cores: 12
+    jobs: 
+      db:
+        endpoints: 
+          - http://asp-perf-db:5001
+      application:
+        endpoints: 
+          - http://asp-perf-lin:5001
+        variables:
+          databaseServer: 10.0.0.103
+      load:
+        endpoints: 
+          - http://asp-perf-load:5001
+
+  aspnet-mono-lin:
+    variables:
+      serverPort: 5000
+      serverAddress: 192.168.1.1
+      cores: 56
+    jobs: 
+      db:
+        endpoints: 
+          - http://asp-mono-load:5001
+      application:
+        endpoints: 
+          - http://asp-mono-lin:5001
+        variables:
+          databaseServer: 192.168.1.1
```

Comment:
This is the ip of the db server from the perspective of the mono-lin one. Is it really 192.168.1.1 ?

Reply:
192.168.1.1 is the private connection between mono-lin and mono-load. Is that the one should be used for talk to the database as well, since for this configuration the mono-load is also the database server.